src/effects/effectbuttonparameterslot.cpp

```diff
@@ -107,3 +109,33 @@ void EffectButtonParameterSlot::slotValueChanged(double v) {
         m_pEffectParameter->setValue(v);
     }
 }
+
+QDomElement EffectButtonParameterSlot::toXML(QDomDocument* doc) const {
```

Comment:
The toXML and loadValuesFromXml calls are not symetric. 
I would prefere to handle EffectXml::ParameterId, in EffectSlot::toXML